app/models/article.rb

```diff
@@ -143,7 +143,7 @@ class Article < ApplicationRecord
            :video_thumbnail_url, :video_closed_caption_track_url, :social_image,
```

Comment:
I think we need the following validators at least:

- co author ids can't contain the user id
- the array needs to be a set essentially (1, 1) can't be added
- ideally we should also check that those IDs actually exist (which was the goal of the foreign key in the other PR)

Reply:
Agreed and updated!